sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.1.0-beta.2 (2021-08-10)
```

Comment:
Need to update the date

Reply:
(2021-08-10) should still be correct if we release today. It's also what we did in the Android hotfix

Reply:
Ahh sorry, my Indian clock showed 8/11 and confused me